README.md

```diff
@@ -224,15 +229,6 @@ nightly at date(< 2023/04/25) | `gcr.io/tpu-pytorch/xla:nightly_3.8_tpuvm_YYYYMM
 
 <br/>
 
-| Version | GPU CUDA 11.7 + Python 3.8 Docker |
-| --- | ----------- |
-| 2.0 | `gcr.io/tpu-pytorch/xla:r2.0_3.8_cuda_11.7` |
-| nightly | `us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:nightly_3.8_cuda_11.7` |
-| nightly at date(>=2023/04/25) | `us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:nightly_3.8_cuda_11.7_YYYYMMDD` |
-| nightly at date(<2023/04/25) | `gcr.io/tpu-pytorch/xla:nightly_3.8_cuda_11.7_YYYYMMDD` |
```

Comment:
you can leave the 11.7 here and only removing the nightly

Reply:
do we want to remove the nightly docker image for `GPU CUDA 11.7 + python 3.8 Docker`?

Reply:
yea removed all 11.7+nightly, and keep the release one.

Reply:
thanks for clarification, updated